tools/incubator.js

```diff
@@ -269,20 +272,29 @@ const Incubator = new Lang.Class({
     },
 });
 
-const USAGE = [
-    'usage: basin-incubator <path_to_input_hatch_directory> <path_to_output_manifest>',
-    '',
-    'Utility that imports a libingester hatch to basin manifest.',
-].join('\n');
+const USAGE = `usage:
+basin-incubator [options] <path_to_input_hatch_directory> <path_to_output_manifest>',
+
+Utility that imports a libingester hatch to basin manifest.
+
+Options:
+  --no-create-sets  don't automatically create set entries for tags
+`;
 
 function main () {
     let argv = ARGV.slice();
+    let create_sets = true;
+    if (argv[0] === '--no-create-sets') {
+        create_sets = false;
+        argv = argv.slice(1);
+    }
+
     let [hatch_dir, manifest_path] = argv;
 
     if (argv.length !== 2)
         fail_with_message(USAGE);
 
-   let incubator = new Incubator(hatch_dir, manifest_path);
+   let incubator = new Incubator(hatch_dir, manifest_path, create_sets);
```

Comment:
We could move this to the hatch_sets.json file... adding a new property, like `ignore-sets: true`?